config/database.yml.example

```diff
@@ -14,12 +14,15 @@
 # Configure Using Gemfile
 # gem 'pg'
 #
+# For details on connection pooling, see rails configuration guide
+# http://guides.rubyonrails.org/configuring.html#database-pooling
```

Comment:
It provide details about connection pooling which I think should stay where it is. Plus it is the default anyway.